### ISSUE TITLE
Documentation error

### ISSUE THREAD

**Issue by juanpasolano** [balderdashy/sails.io.js#69]
The documentation is saying
```
var socket2 = io.sails.connect({
    url: 'https://api.mysite.com'
  });
```
But this will throw a `Uncaught TypeError: self.url.replace is not a function(…)`

This should be 
```
var socket2 = io.sails.connect('https://api.mysite.com')
```

Let me know if I can make a pull request for this to help a little bit

**Comment by sailsbot** [comment]
Thanks for posting, @juanpasolano.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by sgress454** [comment]
Thanks @juanpasolano, patched this up.
